[2025-04-16T20:23:06.499+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T20:23:06.560+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T20:23:04.541989+00:00 [queued]>
[2025-04-16T20:23:06.579+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T20:23:04.541989+00:00 [queued]>
[2025-04-16T20:23:06.581+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-16T20:23:06.605+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): TestConnectionDB> on 2025-04-16 20:23:04.541989+00:00
[2025-04-16T20:23:06.616+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=863) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-16T20:23:06.620+0000] {standard_task_runner.py:72} INFO - Started process 865 to run task
[2025-04-16T20:23:06.620+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'BinanceCryptoAPI', 'TestConnectionDB', 'manual__2025-04-16T20:23:04.541989+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/ETL/BinanceETL.py', '--cfg-path', '/tmp/tmppr9yj2yk']
[2025-04-16T20:23:06.624+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask TestConnectionDB
[2025-04-16T20:23:07.192+0000] {task_command.py:467} INFO - Running <TaskInstance: BinanceCryptoAPI.TestConnectionDB manual__2025-04-16T20:23:04.541989+00:00 [running]> on host 2903c312f900
[2025-04-16T20:23:07.590+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BinanceCryptoAPI' AIRFLOW_CTX_TASK_ID='TestConnectionDB' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T20:23:04.541989+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-16T20:23:04.541989+00:00'
[2025-04-16T20:23:07.592+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-16T20:23:07.731+0000] {base.py:84} INFO - Retrieving connection 'MY_DB'
[2025-04-16T20:23:07.782+0000] {logging_mixin.py:190} INFO - Connection to database successful
[2025-04-16T20:23:07.824+0000] {sql.py:511} INFO - Running statement: SELECT * FROM cryptoInfo, parameters: None
[2025-04-16T20:23:07.854+0000] {sql.py:520} INFO - Rows affected: 0
[2025-04-16T20:23:07.856+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-16T20:23:07.896+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-16T20:23:07.897+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=BinanceCryptoAPI, task_id=TestConnectionDB, run_id=manual__2025-04-16T20:23:04.541989+00:00, execution_date=20250416T202304, start_date=20250416T202306, end_date=20250416T202307
[2025-04-16T20:23:08.087+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T20:23:08.326+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-04-16T20:23:08.351+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
